src/pages/MainStack/Stack.tsx

```diff
@@ -47,7 +47,7 @@ const MainStack = () => {
         <Stack.Screen name={'ReportBakeryStack'} component={ReportBakeryStack} />
         <Stack.Screen name={'NotificationModal'} component={Notification} />
       </Stack.Group>
-      <Stack.Screen name={'Profile'} component={Profile} />
+      <Stack.Screen name="Profile" component={Profile} />
```

Comment:
```suggestion
      <Stack.Screen name={'Profile'} component={Profile} />
```
다른곳과 동일하게 맞췄으면 하는데 어떠신가용?

Reply:
네 그게 좋겠네요 ㅎㅎ 반영할게요!